### PR TITLE
Fix exception for accessing internet by ethernet adapter

### LINKED ISSUE #690: [!] The provided interface "eth0" is invalid!
Description: Program outputs error "[!] The provided interface "eth0" is invalid!"  The AP can be run without a specified --internetinterface, but then the AP doesn't give Internet access to connecting clients that have submitted credentials to the phishing page.  Otherwise, the eth0 interface appears to be configured properly, the machine is connected to the Internet.

Script output: 
wp1
wp2

The script is the most recent version, cloned from Github.

### PR DESCRIPTION
**Pull request by anakin1028**
@blackHatMonkey @sophron 

This should fix #690

### CODE REVIEW

File: wifiphisher/pywifiphisher.py [hunk 1]
```diff
@@ -538,7 +539,8 @@ def start(self):
                 if self.network_manager.is_interface_valid(
                         args.internetinterface, "internet"):
                     internet_interface = args.internetinterface
-                    self.network_manager.unblock_interface(internet_interface)
+                    if pyw.iswireless(internet_interface):
+                        self.network_manager.unblock_interface(internet_interface)
```
Comment (sophron) [inline wifiphisher/pywifiphisher.py]: Can you re-write this part? To keep things clear, it's better not to import `pyric` directly from the main engine but do all the operations through interfaces.py instead.

Comment (sophron) [inline wifiphisher/pywifiphisher.py]: There's no need to check if the interface is wireless at all, right? In any case, you can simply `self.network_manager.unblock_interface(internet_interface)`.

Comment (anakin1028) [inline wifiphisher/pywifiphisher.py]: If the card is not wireless card, it will not support nl80211 driver thus cause pyric raise exception.

Comment (anakin1028) [inline wifiphisher/pywifiphisher.py]: Sure I can add check function in `interface.py`

Comment (anakin1028) [inline wifiphisher/pywifiphisher.py]: Sure I can add check function in `interface.py`

**Comment by blackHatMonkey** [comment]
We should also add this to the test cases for future reference.

**Comment by anakin1028** [comment]
@blackHatMonkey @sophron 
Test case is also included : )

**Pull request merged**